Ports/.port_include.sh

```diff
@@ -58,6 +58,14 @@ function run_export_env() {
     export $1="$2"
 }
 
+function run_fetch_wget() {
```

Comment:
What's the difference between this and run_fetch_web()?

Reply:
wow I didn't know that was there!